src/sentry/static/sentry/app/views/performance/trends/chart.tsx

```diff
@@ -61,31 +60,35 @@ function transformEventStats(data: EventsStatsData, seriesName?: string): Series
   ];
 }
 
-function transformEventStatsSmoothed(data: Series[], seriesName?: string): Series[] {
+function transformEventStatsSmoothed(data: Series[], seriesName?: string) {
+  let minValue = Number.MAX_SAFE_INTEGER;
+  let maxValue = 0;
   const currentData = data[0].data;
   const resultData: SeriesDataUnit[] = [];
-  resultData.push(currentData[0]);
-
-  // Start at 1 so we don't get NaN from the movingAverage
-  for (let i = 1; i < currentData.length; i++) {
-    let value;
-    if (i < AVERAGE_WINDOW) {
-      // A rougher movingAverage for these points, but this way there isn't a gap
-      value = movingAverage(currentData, i, i);
-    } else {
-      value = movingAverage(currentData, i, AVERAGE_WINDOW);
-    }
+
+  const smoothed = smoothTrend(currentData.map(({name, value}) => [Number(name), value]));
```

Comment:
@k-fish Since the resolution is default to 100 for the ASAP function, `smoothed` will always be at most 100 data points. Was this intentional?

It looks like the trends chart can display more than 100 data points.

Reply:
Yes, I'm fine with some oversmoothing as the flow will soon take you to transaction summary with the raw data and the smoothed trend line. If the oversmoothing is too much I can always increase this later. Ideally I'd like to raise the min window size, but that's not an option since it's hardcoded.